clients/drcachesim/common/options.cpp

```diff
@@ -105,7 +105,7 @@ droption_t<bytesize_t> op_chunk_instr_count(
     // We do not support tiny chunks.  We do not support disabling chunks with a 0
     // value, to simplify testing: although we're still having to support generating
     // non-zip files for !HAS_ZLIB/!HAS_ZIP!
-    bytesize_t(1000),
+    bytesize_t(10),
```

Comment:
This was probably only to generate the test trace; should probably be reverted now?

Reply:
It was for the test trace though I thought maybe there's little harm in someone wanting a small value: though it would only be useful for testing.  Reverted back to 1000.